deck.js

```diff
@@ -0,0 +1,263 @@
+const DECK = [
```

Comment:
You should not be hard coding the deck. You could just use the makeDeck function provided. You almost never want to hard code such data because if you make a mistake, you most probably have to change hundreds of lines.